README.md

```diff
@@ -0,0 +1,38 @@
+# Pencil
```

Comment:
Can you please not add `README` yet? Currently the code is hard to understand and we want to simplify it after launching the app. /cc @alaa @herrtreas @23inhouse